app/src/main/java/systems/v/coldwallet/Util/UIUtil.java

```diff
@@ -408,10 +408,13 @@ public static void setExecContractTx(final Activity activity, final Account send
         String time = new Timestamp(timestamp).toString();
         time = time.substring(0, time.indexOf("."));
         timestampTx.setText(time + "\n" + TimeZone.getDefault().getDisplayName());
-
+        String tempAttachment = attachment;
         functionIdTx.setText(String.valueOf(functionId));
-
-        attachmentTx.setText(attachment);
+        try {
+            tempAttachment = new String(Base58.decode(attachment));
+        } catch(Exception e) {
```

Comment:
If the attachment is non-base58 string, it should stop continue and show `Invalid attachment with base58 format` with Toast Text.